concordia/static/scss/action-app.scss

```diff
@@ -82,11 +89,19 @@ main {
     }
 
     .asset-tooltip {
-        background-color: #000;
+        margin: -5px;
+        padding: 14px;
+        background-color: #fff;
+        font-size: 13px;
```

Comment:
This is 1px between `$font-size-base` and `$font-size-small`. Should we be defining it in terms of one of those variables?

The same question applies to the `line-height` below, which is just .03 smaller than `$line-height-base`